### ISSUE TITLE
[Feat] No Products on any shop

### ISSUE THREAD

**Issue by inkerton** [mdazfar2/Ezyshop#1010]
### Description

I would like to add some dummy products to these shops.

Please assign me this issue I will add 5 products to each shop.

### Screenshots

![image](https://github.com/user-attachments/assets/6b7e5ed5-5a63-4e12-8009-6cc34da060d2)

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/mdazfar2/Ezyshop/blob/main/CONTRIBUTING.md)
- [X] I'm a GSSoC'24-Extd contributor
- [X] I'm a Hacktoberfest'24 contributor
- [X] I am willing to work on this issue (optional)

**Comment by mdazfar2** [comment]
Hey @inkerton products are added by their shop owners only. I think you are not familiar with the project structure and how it works. Please read the [README.md](https://github.com/mdazfar2/Ezyshop/blob/main/README.md) file for understanding and then raise an issue. I am closing the issue for now. Please raise another issue, which I am happy to assign to you, but make sure the issue is valid and addresses something important on our website. Otherwise, invalid issues will be closed.

In short and concise, when logged in as a seller, you get the functionality to add your own shop and products. Hope you understand now.




 Thank you!
